e2e/assets/single-cluster/helm.yaml

```diff
@@ -0,0 +1,15 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: helm
+spec:
+  repo: https://github.com/rancher/fleet-examples
+  paths:
+  - single-cluster/helm
+  targets:
+  - clusterSelector:
+      matchExpressions:
+      - key: provider.cattle.io
+        operator: NotIn
+        values:
+        - harvester
```

Comment:
don't think this is required in standalone fleet testing

Reply:
Right. On the other hand it's good to see that this gets deployed nevertheless on a single-cluster.